#Configuration file for laser shade occlusion manager
{
    use_polygon_occlusion_manager: true,
    occlusion_manager_visualization_enabled: true,
    occlusion_manager_time_uncertainty_factor: 1.3,
    occlusion_manager_max_range: 20,
    occlusion_self_occlusion_distance: 0.50,
    occlusion_geodesics_motion_variance: 1.0,
    occlusion_geodesics_total_cut_off: 0.1,
    occlusion_geodesics_plausibility_cut_off: 0.01,
    occlusion_geodesics_inertia_variance: 0.1,
    max_misses_before_deletion: 8,
    max_misses_before_deletion_of_mature_track: 15,
    occlusion_manager_subscriber_queue_size: 100,
    occlusion_manager_sync_buffer_size: 100,
    occlusion_geodesics_use_detection_probability: 0,
    occlusion_allowed_duration_for_reappearance: 1.0,

}
